libraries/GCS_MAVLink/GCS.h

```diff
@@ -886,6 +888,28 @@ class GCS
     virtual MAV_TYPE frame_type() const = 0;
     virtual const char* frame_string() const { return nullptr; }
 
+    struct statustext_t {
+        uint8_t                 bitmask;
+        mavlink_statustext_t    msg;
+        uint16_t                entry_created_ms;
+    };
+    class StatusTextQueue : public ObjectArray<statustext_t> {
```

Comment:
Why not just add an ObjectArray_TS to utility like we had to do for ringbuffers?

Reply:
Pretty sure this PR predates the `_TS` objectarray.

When we need a second thread-safe objectarray we can create it then.